### PR TITLE
Fix some minor documentation bugs

### LINKED ISSUE #3483: Rosetta ReadMe incomplete sentence
### Description

The Rosetta [ReadMe](https://github.com/hashgraph/hedera-mirror-node/blob/main/docs/rosetta/README.md#genesis-balance-file) has an incomplete sentence at the end of the first paragraph in the Genesis Balance File section: 

Screen Shot 2022-03-30 at 11 50 48 AM

### Steps to reproduce

1. Navigate to the documentation at https://github.com/hashgraph/hedera-mirror-node/blob/main/docs/rosetta/README.md#genesis-balance-file

### Additional context

_No response_

### Hedera network

other

### Version

v0.53.0

### Operating system

_No response_

### LINKED ISSUE #4404: Slightly misleading REST API doc
### Description

The description of the `reward_rate_start` property returned by `/api/v1/network/nodes ` can be misleading since the value is not the total amount of tinybars earned but the amount of tinybars earned per hbar staked.

### Steps to reproduce

Swagger doc at: http://previewnet.mirrornode.hedera.com/api/v1/docs/#/network/getNetworkNodes

### Additional context

Screenshot 2022-09-02 at 19 16 41

### Hedera network

mainnet

### Version

 0.63.0

### Operating system

_No response_

### LINKED ISSUE #4406: Typo in the REST API doc for BlocksResponse
### Description

The API doc for BlocksResponse says:
Screenshot 2022-09-02 at 23 46 39
but the name of the first field of the data structure is actually `blocks`.

### Steps to reproduce

http://previewnet.mirrornode.hedera.com/api/v1/docs/#/blocks/listBlocks

### Additional context

_No response_

### Hedera network

previewnet

### Version

 0.63.0

### Operating system

_No response_

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:

* Fix incomplete sentence in Rosetta README
* Fix incorrect field name in list blocks OpenAPI
* Fix incorrect `reward_rate_start` description in OpenAPI
* Fix incorrect use of `type: number` with integer formats

**Related issue(s)**:

Fixes #3483
Fixes #4404
Fixes #4406

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by edwin-greene — DISMISSED** [review]
Looks good.

File: hedera-mirror-rest/api/v1/openapi.yml [hunk 1]
```diff
@@ -2388,7 +2412,9 @@ components:
         modified_timestamp:
           $ref: '#/components/schemas/TimestampNullable'
         serial_number:
-          type: number
+          example: 1
+          format: int32
```
Comment (xin-hedera) [inline hedera-mirror-rest/api/v1/openapi.yml:2416]:
```suggestion
          format: int64
```

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/api/v1/openapi.yml (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-rest/api/v1/openapi.yml]: Thanks, fixed

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**